pkg/persistence/db.go

```diff
@@ -108,7 +109,7 @@ func (c *conn) ClaimKey(oneTimeCode string, appPublicKey []byte) ([]byte, error)
 
 const maxOneTimeCode = 1e8
 
-func (c *conn) NewKeyClaim(region, originator string) (string, error) {
+func (c *conn) NewKeyClaim(region, originator, hashId string) (string, error) {
```

Comment:
likewise, `hashID`. I won't call them all out individually.

Reply:
Fixed in #57